servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,40 @@
 ---
 title: ServiceControl in Practice
-summary: Details how to think about the use of ServiceControl in production environments
+summary: Considerations for configuring ServiceControl for use in production environments
 tags:
 - ServiceControl
 ---
 
-ServiceControl is a tool that can accommodate many situations. Features can be enabled or disabled depending on the needs of the environment. Each feature is valuable in certain contexts but also comes with a cost in terms of resource usage and performance. This guidance is provided to provoke thought about why any given feature or plugin might be valuable in a system environment and to help decision makers make informed choices about how to get the best out of ServiceControl without providing solutions to specific scenarios that consider choices around hardware, transport or persistence.
 
-For example, in a development environment the [Debug Session](/servicecontrol/plugins/debug-session.md) and [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugins provide detailed information to support problem analysis. Particular advise that customers not use these plugins outside of a development environment.
+ServiceControl has many capabilities which can be extended by adding optional plugins into the endpoints of the system being monitored. Each capability and plugin can provide valuable information but also comes with a cost in terms of resource usage and performance. There are other factors that can influence the resource and performance profile of ServiceControl including hardware, message throughput, and number of endpoints. These factors can vary greatly between environments. Capabilities and plugins that provide value in one environment may have a negative impact if included in another environment.
```

Comment:
> Would it make sense to list what capabilities SC has out of the box, without plugins?

Interesting...if there are no plugins installed on endpoints does it do anything?

> Hardware, (expected/real?) message throughput (load?)

When I was initially writing this I was trying to encourage people to gather their own metrics through performance testing. I don't think we should provide numbers, customers should seek their own and do their own analysis.